src/YunInternet/Libvirt/Configuration/Domain/Device/Disk/BackingStore.php

```diff
@@ -0,0 +1,54 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * Date: 19-2-17
+ * Time: 上午1:20
+ */
+
+namespace YunInternet\Libvirt\Configuration\Domain\Device\Disk;
+
+use YunInternet\Libvirt\XMLImplement\SimpleXMLImplement;
+use YunInternet\Libvirt\XMLImplement\SingletonChild;
+
+/**
+ * Class backingStore
+ * @method XMLElementContract source()
+ * @method XMLElementContract format()
+ * @package YunInternet\Libvirt\Configuration\Domain\Device\Disk
+ */
+class BackingStore extends SimpleXMLImplement
+{
+    use SingletonChild;
+
+    public function active()
```

Comment:
Hi, this method, I think naming as isActive will be better.

Reply:
Just like this function: https://libvirt.org/php/api-reference.html#libvirt_domain_is_active